src/development/tests.md

```diff
@@ -242,22 +260,18 @@ The steps from the overview are elaborated on here.
            debug!("Starting receive of length {}", rx_len);
            let (err, _opt) = self.device.receive_buffer(rx_buffer, rx_len);
            if err != ReturnCode::SUCCESS {
-               panic!(
+               debug!(
                    "Calling receive_buffer() in virtual_uart test failed: {:?}",
                    err
                );
+               self.client.map(|client| {
+                   client.done(Err(CapsuleTestError::ErrorCode(ErrorCode::FAIL)));
+               });
```

Comment:
Shouldn't this also send an `Ok()` if it wasn't an error?

Reply:
Added the OK case